includes/wc-core-functions.php

```diff
@@ -1408,6 +1408,54 @@ function wc_get_logger() {
 }
 
 /**
+ * Dump an expression safely.
+ *
+ * Some server environments blacklist some debugging functions. This function provides a safe way to
+ * turn an expression into a printable, readable form while protecting from errors.
+ *
+ * @since 2.8
+ * @param mixed $expression The expression to be printed.
+ * @return string|bool Result of printing expression. False if no alternatives are available.
+ */
+function wc_safe_dump( $expression ) {
+	$alternatives = array(
+		array( 'func' => 'print_r', 'args' => array( $expression, true ) ),
+		array( 'func' => 'var_dump', 'args' => array( $expression, true ) ),
```

Comment:
😱  Thanks for catching that. We can probably just drop `var_dump` as an alternative.